docker-compose.yml

```diff
@@ -68,13 +68,16 @@ services:
       - gat-node
       - gat-db
   db:
-    image: "postgres:15.3-alpine"
+    build: 
+      dockerfile: Dockerfile
+      context: openshift/templates/images/postgres
     restart: always
     environment:
       POSTGRES_PASSWORD: *POSTGRES_PASSWORD
       POSTGRES_USER: *POSTGRES_USER
     volumes:
       - database:/var/lib/postgresql/data
+      - ./openshift/templates/images/postgres/postgresql.conf:/tmp/postgresql.conf
```

Comment:
so that's how we store the config for postgres so it can use that plugin on openshift. cool!